File: JS/Car.js
// Creación de clases en JavaScript
function Car(license, driver/*Al colocar las variables aqui, creamos el método constructor*/ ) {
    this.id; // Atributos de la clase
    this.license = license; // Atributos de la clase, Asigna directamente las variables locales a las globales
    this.driver = driver; // Atributos de la clase, Asigna directamente las variables locales a las globales
    this.passenger; // Atributos de la clase
}

Car.prototype.printDataCar = function () {
    //Método de la clase
    console.log(this.driver)
    console.log(this.driver.name)
    console.log(this.driver.document)
}

File: Python/car.py
from account import Account # Importar clase
# Creación de clases en python
class Car:
    id          =int # Atributos de la clase
    license     =str # Atributos de la clase
    driver      = Account("","") # Atributos de la clase, Atributo que hereda de la clase Account
    passenger   =str # Atributos de la clase
    
    def __init__(self, license, driver):
        # Método constructor
        self.license=license
        self.driver=driver